plugins/feeds/public/lolbas.py

```diff
@@ -126,3 +163,5 @@ def format_commands(self, commands: list[dict[str, str]]) -> str:
 
 
 taskmanager.TaskManager.register_task(LoLBAS)
+taskmanager.TaskManager.register_task(LoLBAS)
+taskmanager.TaskManager.register_task(LoLBAS)
```

Comment:
Duplicate registration?

Reply:
Fixed in 1c5cc8f0abd23d6e1dab6589460acad05faa5452